source/_posts/2018-05-30-home-assistant-security.markdown

```diff
@@ -0,0 +1,250 @@
+---
+layout: post
+title: "Home Assistant security"
+description: "Open ports of a Hass.io installation with different add-ons."
+date: 2018-05-30 00:01:00
+date_formatted: "May 30, 2018"
+author: Fabian Affolter
+author_twitter: fabaff
+comments: true
+categories: Release-Notes
+og_image: /images/blog/2018-05-security/social.png
+---
+
+
+There were a couple of threads in our forum with scary titles like "[I have been Hacked](https://community.home-assistant.io/t/i-have-been-hacked/53723)", "[HA security and hacking](https://community.home-assistant.io/t/ha-security-and-hacking/54322)" or "[I got hacked](https://community.home-assistant.io/t/i-got-hacked/54083)". Of course, these long discussions contain some rumors, speculations and assumptions. 
+
+Just to be clear: We, the Home Assistant Developers, take security, privacy and integrity very serious. 
+
+As a large number of users are running [Hass.io](/hassio/), I'm going to use a Raspberry Pi 3 B and Hass.io 0.70.0 to show how Home Assistant looks from the network side. This is not a full blown investigation, just a quick overview to give you some closure.
+
+The IP address of the Home Assistant machine is 192.168.0.215. The system which is the source of the scans is a machine running Fedora 27 and Nmap 7.60 is used to preform the port scans. Both systems are in the same network. Internet access is provided by a OPNsense firewall which is connected to the cable modem of my ISP.
+
+There is an `api_password` set but otherwise Hass.io is running with a default configuration that is created on the first launch.
+
+The default port of Home Assistant is 8123. This is the port where the [`frontend`](/components/frontend/) and the [`API`](/components/api/) are served. Both are depending on the [`http`](/components/http/) component which contains the capability to adjust the settings like `server_host` or `server_port`.
+
+In the next section we are going to test Hass.io with the three most popular add-ons.
```

Comment:
Why would we do this? If we want to address concerns, we should do that. 

We should talk about how Home Assistant Cloud does not require you to open up your instance to the internet. How Home Assistant Cloud will setup a secure connection to a remote server that will only allow Google Assistant and Amazon Alexa limited access to control the devices the user wants to allow to control.

We should talk about how we're working on a user and permission system to allow locking things down.